File: python/7 kyu/184.py
# https://www.codewars.com/kata/58aa68605aab54a26c0001a6/train/python

# Passed

def distinct_digit_year(year):
    is_distinct = False
    while not is_distinct:
        year += 1
        is_distinct = len(set(str(year))) == len(str(year))
        
    return year

output = distinct_digit_year(1987)
print(output)

File: nodejs/7 kyu/184.js
// https://www.codewars.com/kata/554b4ac871d6813a03000035/train/javascript

// Passed

function highAndLow(numbers) {
  let allNumbers = numbers.split(" ").map((strNum) => Number(strNum));
  let maximumNum = Math.max(...allNumbers);
  let minimumNum = Math.min(...allNumbers);
  let result = [maximumNum, minimumNum].join(" ");
  return result;
}

const output = highAndLow("8 3 -5 42 -1 0 0 -9 4 7 4 -4");
console.log(output);